packages/math-input/src/components/keypad/__tests__/keypad-v2.cypress.ts

```diff
@@ -0,0 +1,61 @@
+import renderSingleKeypad from "../../../../../../testing/render-keypad-with-cypress";
+
+type StringDictionary = {[key: string]: string};
+type TabArray = StringDictionary[];
+const tabs: TabArray = [
+    {name: "Operators", specialButton: "EXP_2"},
+    {name: "Extras", specialButton: "PI"},
+
+    {name: "Geometry", specialButton: "COS"},
+    {name: "Numbers", specialButton: "NUM_7"},
+];
+
+describe("Keypad v2", () => {
+    it("renders", () => {
+        renderSingleKeypad((key) => {});
+    });
+
+    tabs.forEach((tab) => {
+        it(
+            "switches to the " +
+                tab["name"] +
+                " tab when we press on the " +
+                tab["name"] +
+                " button",
+            () => {
```

Comment:
```suggestion
        it(`switches to the correct tab: ${tab.name}`, () => {
```

I think this might be enough for this test